lighthouse-docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM node:9-slim
```

Comment:
are we sure we want to use node 9? and not node 8?

Reply:
No pref really. Node 9 produces a slightly smaller image.